### ISSUE TITLE
Multiple (I tried with 2) Lottie animation in one screen lags for 3 - 4 seconds

### ISSUE THREAD

**Issue by begouravdev** [lottie-react-native/lottie-react-native#867]
### Description

I want to play two animations on one screen but that's lagging for around 3 - 4 seconds on iPhone 12 pro max simulator
But if I play any one of them perfectly working

### Steps to Reproduce
put any two animation in serial
<AnimatedLottieView
          source={LOTTIE_ANIMATION.HANDSHAKE}
          style={cs.animation}
          loop={false}
          autoPlay
          onAnimationFinish={(status) => setAnimationComplete(!status)}
        />
<AnimatedLottieView
          source={LOTTIE_ANIMATION.CONFETTI}
          style={cs.animation}
          loop={false}
          autoPlay
          onAnimationFinish={(status) => setAnimationComplete(!status)}
        />

**Expected behavior:** 
Should be played

**Actual behavior:**
Not playing

### Versions

lottie-react-native 5.0.1
lottie-ios 3.2.3

**Comment by begouravdev** [comment]
LOGS:

This can break usage such as persisting and restoring state. This might happen if you passed non-serializable values such as function, class instances etc. in params. If you need to use components with callbacks in your options, you can use 'navigation.setOptions' instead. See https://reactnavigation.org/docs/5.x/troubleshooting#i-get-the-warning-non-serializable-values-were-found-in-the-navigation-state for more details. 
BaseNavigationContainer@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:210528:28
ThemeProvider@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:214871:21
NavigationContainer@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:214777:26
Router@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:266055:36
LoadingContainer@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:203233:24
Connect(LoadingContainer)@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:180984:43
PersistGate@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:457162:22
CustomerServiceProvider@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:413640:24
TrackingProvider@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:175953:24
Connect(TrackingProvider)@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:180984:43
IAPProvider@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:426642:24
Connect(IAPProvider)@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:180984:43
Provider@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:180502:21
App@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:134169:36
WrapperComponent@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:154718:36
RCTView
View
RCTView
View
AppContainer@http://localhost:8081/index.bundle?platform=ios&dev=true&minify=false&modulesOnly=false&runModule=true&app=com.breeze.mobile.app:77362:36
 WARN  Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`

**Comment by MuhammadAbdullah54321** [comment]
I am facing the same issue.
I have a screen where I am using a component with 5 Lottie emojis. 
My whole screen loads very slow when I navigate to it. Screen stucks in the beginning and then after like 2-3 secs everything loads.
I have checked by commenting out code. Its the component with Lottie emojis that is causing this delay.

I have tried both with and without "HARDWARE" renderMode

**Comment by matinzd** [comment]
Can any of you provide us a repro?

**Comment by MuhammadAbdullah54321** [comment]
Actually I cannot provide you the repo because it's private one of client.

I can explain the issue and how I solved it.
So issue in my case was that lottie JSON files were too big. All 5 lottie files were well above 1 MB some were like 1.2 MB and some were like 1.6 MB. 

Actually Lottie emoji animations were created in very high resolution. The original resolution size was: 3840*2160
So we decreased the resolution because this was overkill for mobile screen. We brought it down to 160*160 and so size was considerably reduced. It came down to somewhere around 100kb. Result of animations on screen was still pretty good and there was no noticeable difference in quality of animations. 
Doing this solved the problem and now screen was loading pretty fast. 

We are actually using Adobe After Effects to create those emoji animations and then converting it to Lottie JSON. I think that conversion is not very optimal. Because I have tried similar kind of available emoji animations on Lottie website ... they were still more faster then ours because their size was much smaller ... like 18 kb to 30 kb. But ours are still above 100 Kb/animation
